content/en/docs/features/sealed-secrets.md

```diff
@@ -8,4 +8,135 @@ tags:
+automatically.
+In addition to generating the secret string, you must also upload the resource
+to your KBS.
+{{% /alert %}}
+
+### Adding a sealed secret to Kubernetes
+
+Create a secret from your secret string using `kubectl`.
+```bash
+kubectl create secret generic sealed-secret --from-literal='secret=sealed.fakejwsheader.ewogICAgInZlcnNpb24iOiAiMC4xLjAiLAogICAgInR5cGUiOiAidmF1bHQiLAogICAgIm5hbWUiOiAia2JzOi8vL2RlZmF1bHQvc2VhbGVkLXNlY3JldC90ZXN0IiwKICAgICJwcm92aWRlciI6ICJrYnMiLAogICAgInByb3ZpZGVyX3NldHRpbmdzIjoge30sCiAgICAiYW5ub3RhdGlvbnMiOiB7fQp9Cg==.fakesignature'
+```
+
+Note that `--from-literal` takes a mapping of secret keys to secret values.
+The secret value should be the string generated in the previous step.
+The secret key can be whatever you want, but make sure to use the same one in future steps.
+
+### Deploying a sealed secret to a confidential workload
+
+You can add your sealed secret to a workload yaml file.
+
+You can expose your sealed secret as an environment variable.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sealed-secret-pod
+spec:
+  runtimeClassName: kata-qemu-coco-dev
+  containers:
+  - name: busybox
+    image: quay.io/prometheus/busybox:latest
+    imagePullPolicy: Always
+    command: ["echo", "$PROTECTED_SECRET"]
+    env:
+    - name: PROTECTED_SECRET
+      valueFrom:
+        secretKeyRef:
+          name: sealed-secret
+          key: secret
+```
+
+You can also expose your secret as a volume.
+```yaml 
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-test-pod-cc
+spec:
+  runtimeClassName: kata
+  containers:
+  - name: busybox
+    image: quay.io/prometheus/busybox:latest
+    imagePullPolicy: Always
+    command: ["cat", "/sealed/secret-value/secret"]
+    volumeMounts:
+        - name: sealed-secret-volume
+          mountPath: "/sealed/secret-value"
+  volumes:
+    - name: sealed-secret-volume
+      secret:
+        secretName: sealed-secret
+```
+{{% alert title="Note" color="primary" %}}
+Currently sealed secret volumes must be mounted
+in the `/sealed` directory.
+{{% /alert %}}
+
+## Advanced
+
+### Envelope Secrets
+
+You can also create envelope secrets.
+With envelope secrets, the secret itself is included in the secret
+(unlike a vault secret, which is just a pointer to a secret).
+In an envelope secret, the secret value is wrapped and can be unwrapped
+by a KMS.
+This allows us to support models where the key for unwrapping secrets
+never leaves the KMS.
+It also decouples the secret from the KBS.
+
+We currently support two KMSes for envelope secrets.
+See specific instructions for [aliyun kms](https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/docs/kms-providers/alibaba.md)
+and [eHSM](https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/docs/kms-providers/ehsm-kms.md).
+
+
+### Integrity Protection
+
+Sealed secrets do not currently support integrity protection.
```

Comment:
should we move that up (to L64) into an alert-block, so readers get that information right next to the fake-header/signature string?

Reply:
Good idea. I move this to a note right after that example.